### PR TITLE
new resource: `azurerm_kubernetes_fleet_member`

### PR DESCRIPTION
**Pull request by ms-henglu**
image

### CODE REVIEW

File: internal/services/containers/kubernetes_fleet_member.go [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+				}
+			}
+			if !response.WasNotFound(existing.HttpResponse) {
+				return metadata.ResourceRequiresImport(r.ResourceType(), id)
+			}
+
+			payload := fleetmembers.FleetMember{
+				Properties: &fleetmembers.FleetMemberProperties{
+					ClusterResourceId: config.KubernetesClusterId,
+					Group:             pointer.To(config.Group),
```
Comment (magodo) [inline internal/services/containers/kubernetes_fleet_member.go:97]: Should we conditionally set this?

File: internal/services/containers/kubernetes_fleet_member_test.go [hunk 2]
```diff
@@ -0,0 +1,179 @@
+package containers_test
```
Comment (magodo) [inline internal/services/containers/kubernetes_fleet_member_test.go:1]: Can we add a test case to demonstrate the multi-cluster member case, albeit the update test case?

**Review by magodo — CHANGES_REQUESTED** [review]
Thanks for this PR!
I have left some additional mostly minor comments that once addressed this should be good to merge 👍

**Comment by magodo** [comment]
LGTM!

**Comment by stephybun** [comment]
@ms-henglu generation of this resource with https://github.com/hashicorp/pandora/pull/3416 should be unblocked by https://github.com/hashicorp/pandora/pull/3757.

I'm going to close this PR for the moment to prevent this from being merged accidentally. We can reopen this if there are other blocking issues.

**Pull request closed without merging**